README.md

```diff
@@ -187,7 +187,7 @@ Parameter Based
   * `constructor(cronTime, onTick, onComplete, start, timezone, context, runOnInit)` - Of note, the first parameter here can be a JSON object that has the below names and associated types (see examples above).
     * `cronTime` - [REQUIRED] - The time to fire off your job. This can be in the form of cron syntax or a JS [Date](https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/Date) object.
     * `onTick` - [REQUIRED] - The function to fire at the specified time.
```

Comment:
I think it would be also useful to update the documentation for `onTick` too. To let everyone know that it receives onComplete as a parameter (and you can optionally use it as a callback)